datadog.go

```diff
@@ -125,9 +125,7 @@ func sanitizeMetricName(namespace string, v *view.View) string {
 func viewSignature(namespace string, v *view.View) string {
 	var buf strings.Builder
 	buf.WriteString(sanitizeMetricName(namespace, v))
-	for _, k := range v.TagKeys {
-		buf.WriteString("_" + k.Name())
-	}
+
```

Comment:
Looks like this will be a breaking change.

So, it's probably better to offer an option to disable it and add a deprecation warning, but keep the default behaviour as it. That will let users have time to react before we remove it completely.